src/css/buttons.css

```
@@ -288,4 +267,167 @@
   color: var(--color-moon-200);
 }
 
+/* Iconlabel button */
+
+.a-btn--iconlabel .a-icon {
+  background-color: var(--color-space-100);
+  margin: -4px 9px -3px 0;
+  transition: all 0.3s ease;
+}
+
+.a-btn--iconlabel--right .a-icon {
```

I think for compound modifiers, BEM indicates something like:

```suggestion
.a-btn--iconlabel-right .a-icon {
```

---

Done on 6a35751